core/ReportRenderer/Html.php

```diff
@@ -131,7 +131,6 @@ public function renderReport($processedReport)
                     $processedReport['segment']
                 );
                 $reportView->assign("generatedImageGraph", base64_encode($staticGraph));
-                unset($generatedImageGraph);
```

Comment:
The variable `$generatedImageGraph` is not set anywhere, so no need to unset it.